1636:
Traceback (most recent call last):
  File "S:/Engineering/Jake/mc-missing-data/asset_accession_form_scraping_script.py", line 405, in <module>
    combined_asset_info.append(gather_data())
  File "S:/Engineering/Jake/mc-missing-data/asset_accession_form_scraping_script.py", line 267, in gather_data
    calibration_maintenance_tab = driver.find_element_by_id('formTabsnav2')
  File "C:\ProgramData\Anaconda3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 359, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\ProgramData\Anaconda3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 966, in find_element
    'value': value})['value']
  File "C:\ProgramData\Anaconda3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\ProgramData\Anaconda3\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"id","selector":"formTabsnav2"}
  (Session info: chrome=70.0.3538.110)
  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 6.1.7601 SP1 x86_64)

 Message: no such element: Unable to locate element: {"method":"id","selector":"formTabsnav2"}
  (Session info: chrome=70.0.3538.110)
  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 6.1.7601 SP1 x86_64)

